napari/layers/shapes/shape_models/shape.py

```diff
@@ -1,33 +1,38 @@
 from abc import ABC, abstractmethod
-import numpy as np
 from copy import copy
+
+import numpy as np
 from vispy.color import Color
+
 from ..shape_utils import (
     triangulate_edge,
     triangulate_face,
     is_collinear,
     poly_to_mask,
     path_to_mask,
 )
+from ...utils.colormaps.standardize_colors import transform_color
 
 
 class Shape(ABC):
-    """Base class for a single shape
+    """Base class for a single shape.
 
     Parameters
     ----------
     data : (N, D) array
         Vertices specifying the shape.
     edge_width : float
         thickness of lines and edges.
-    edge_color : str | tuple
-        If string can be any color name recognized by vispy or hex value if
-        starting with `#`. If array-like must be 1-dimensional array with 3 or
-        4 elements.
-    face_color : str | tuple
-        If string can be any color name recognized by vispy or hex value if
-        starting with `#`. If array-like must be 1-dimensional array with 3 or
-        4 elements.
+    edge_color : str or array-like
+        Color of the shape marker border. Strings can be color names or hex
+        values (starting with `#`). Array-like should have 3 or 4 elements
```

Comment:
Shape color has always been global per shape - i.e. all the edges are "red" this now makes it sound like you could give a different color to each edge, I'm not sure we want to support that from an API complexity standpoint, though we certainly could. Was that change intentional?

Particularly I'm not sure if: `If array-like must be 1-dimensional array` -> `Array-like should have 3 or 4 elements per row` is a change we want to make